### ISSUE TITLE
Net Plugin not sending any data on Windows Server 2012

### ISSUE THREAD

**Issue by ConfiDev** [influxdata/telegraf#1949]
Windows Server 2012 R2 Standard / Windows 10 Pro (both tested with same config)
Telegraf 1.0.1

I'm using the Net Input Plugin. The plugin is getting detected and started as input plugin but its not sending any data to the InfluxDb. The other stats are getting send to InfluxDb, e.g. cpu, mem.
I started the plugin without any additional configurations. (the interfaces option is not used)

The exact same configuration is working on our linux machine.

Console Output:

```
C:\Program Files\Telegraf>telegraf.exe --debug --config telegraf.conf
2016/10/25 15:10:59 Attempting connection to output: influxdb
2016/10/25 15:11:00 Successfully connected to output: influxdb
2016/10/25 15:11:00 Starting Telegraf (version 1.0.1)
2016/10/25 15:11:00 Loaded outputs: influxdb
2016/10/25 15:11:00 Loaded inputs: cpu diskio mem net
2016/10/25 15:11:00 Tags enabled: host=123
2016/10/25 15:11:00 Agent Config: Interval:10s, Debug:true, Quiet:false, Hostname:"123", Flush Interval:10s
2016/10/25 15:11:10 Input [cpu] gathered metrics, (10s interval) in 0
2016/10/25 15:11:10 Input [mem] gathered metrics, (10s interval) in 0
2016/10/25 15:11:10 Input [net] gathered metrics, (10s interval) in 7.0004ms
2016/10/25 15:11:10 Input [diskio] gathered metrics, (10s interval) in 290.9992ms
2016/10/25 15:11:20 Input [cpu] gathered metrics, (10s interval) in 0
2016/10/25 15:11:20 Input [mem] gathered metrics, (10s interval) in 999.5µs
2016/10/25 15:11:20 Input [net] gathered metrics, (10s interval) in 3.9982ms
2016/10/25 15:11:20 Output [influxdb] buffer fullness: 6 / 10000 metrics. Total gathered metrics: 6. Total dropped metrics: 0.
2016/10/25 15:11:20 Output [influxdb] wrote batch of 6 metrics in 4.0001ms
2016/10/25 15:11:20 Input [diskio] gathered metrics, (10s interval) in 282.9985ms
2016/10/25 15:11:30 Input [mem] gathered metrics, (10s interval) in 0
2016/10/25 15:11:30 Input [cpu] gathered metrics, (10s interval) in 0
2016/10/25 15:11:30 Input [net] gathered metrics, (10s interval) in 5.9837ms
2016/10/25 15:11:30 Output [influxdb] buffer fullness: 5 / 10000 metrics. Total gathered metrics: 11. Total dropped metrics: 0.
2016/10/25 15:11:30 Output [influxdb] wrote batch of 5 metrics in 17.0006ms
2016/10/25 15:11:30 Input [diskio] gathered metrics, (10s interval) in 282.9991ms
2016/10/25 15:11:40 Input [mem] gathered metrics, (10s interval) in 0
2016/10/25 15:11:40 Input [cpu] gathered metrics, (10s interval) in 0
2016/10/25 15:11:40 Input [net] gathered metrics, (10s interval) in 4.0056ms
2016/10/25 15:11:40 Output [influxdb] buffer fullness: 5 / 10000 metrics. Total gathered metrics: 16. Total dropped metrics: 0.
2016/10/25 15:11:40 Output [influxdb] wrote batch of 5 metrics in 1.9995ms
2016/10/25 15:11:40 Input [diskio] gathered metrics, (10s interval) in 285.003ms
2016/10/25 15:11:50 Input [mem] gathered metrics, (10s interval) in 0
2016/10/25 15:11:50 Input [cpu] gathered metrics, (10s interval) in 0
2016/10/25 15:11:50 Input [net] gathered metrics, (10s interval) in 4.9553ms
2016/10/25 15:11:50 Output [influxdb] buffer fullness: 5 / 10000 metrics. Total gathered metrics: 21. Total dropped metrics: 0.
```

**Comment by sparrc** [comment]
can you run a test output?: `telegraf.exe --debug --config telegraf.conf --test`

**Comment by ConfiDev** [comment]
No Data fround:
```
C:\Program Files\Telegraf>telegraf.exe --debug --config telegraf.conf --test
* Plugin: net, Collection 1
* Plugin: cpu, Collection 1
* Plugin: cpu, Collection 2
> cpu,host=123 usage_guest=0,usage_guest_nice=0,usage_idle=99.21875,usage_iowait=0,usage_irq=0,usage_nice=0,usage_softirq=0,usage_steal=0,usage_system=0.1953125,usage_user=0.
5859375 1477404045000000000
* Plugin: diskio, Collection 1
> diskio,host=123,name=C: io_time=0i,read_bytes=0i,read_time=0i,reads=0i,write_bytes=10012i,write_time=0i,writes=0i 1477404046000000000
> diskio,host=123,name=E: io_time=0i,read_bytes=0i,read_time=0i,reads=0i,write_bytes=0i,write_time=0i,writes=0i 1477404046000000000
> diskio,host=123,name=P: io_time=0i,read_bytes=0i,read_time=0i,reads=0i,write_bytes=0i,write_time=0i,writes=0i 1477404046000000000
* Plugin: mem, Collection 1
> mem,host=123 active=0i,available=3495325696i,available_percent=10.174099885019155,buffered=0i,cached=0i,free=0i,inactive=0i,total=34355134464i,used=30859808768i,used_percen
t=89.82590011498084 1477404046000000000
```

**Comment by sparrc** [comment]
thanks @ConfiDev, I've tracked this down to a bug in the windows build of a system information library we're using. I've submitted a PR here: https://github.com/shirou/gopsutil/pull/275 to fix that, so hopefully we can have that merged and in Telegraf by version 1.2
